Dockerfile

```diff
@@ -206,6 +206,25 @@ RUN apt-get -q install -y \
     pkg-config \
     && rm -r /var/lib/apt/lists/*
 
+#install flutter
+ENV FLUTTER_HOME=/root/flutter
+RUN curl -o flutter_linux_2.8.1-stable.tar.xz https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_2.8.1-stable.tar.xz
+RUN tar xf flutter_linux_2.8.1-stable.tar.xz
+RUN mv flutter ${FLUTTER_HOME}
+RUN rm flutter_linux_2.8.1-stable.tar.xz
```

Comment:
These RUN commands should be merged into a single RUN to minimize the number of docker layers and docker size. Unless there is a reason these need to be ran as separately?

Reply:
Definitely true. Had to separate to help debugging but now, no good reason to keep it that way. I fixed it. Thank you !